package.json

```diff
@@ -31,7 +31,7 @@
     "esbuild": "0.17.3",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.5.0",
-    "obsidian": "latest",
+    "obsidian": "^1.5.7-1",
```

Comment:
I pulled this PR locally and ran npm install, `npm list obsidian` gives me 1.5.7-1 correctly. However, I see this when I run the plugin
<img width="730" alt="image" src="https://github.com/logancyang/obsidian-copilot/assets/4860545/5939d0dc-bfec-4500-b6a5-ccbd8d9e9b68">

Can you reproduce it? This is weird.

Reply:
You need the last obsidian version

Reply:
@Lisandra-dev you mean I need to update the Obsidian app itself, not just the API version?

Reply:
OK I see, I updated the app and it's working now. But hesitant to make this breaking change just for frontmatter removal, it forces all users to upgrade the Obsidian app. Can this just be a simple util we can implement by ourselves?

Reply:
To be honest, I think it's the simpler and safer way to remove frontmatter. But we can use a simple regex to remove it. I can change it ASAP.